src/testUtils.ts

```diff
@@ -166,7 +166,7 @@ export function goTest(testconfig: TestConfig): Thenable<boolean> {
 			outputChannel.show(true);
 		}
 
-		let buildTags: string = testconfig.goConfig['buildTags'];
+		let testTags: string = (testconfig.goConfig.has("testTags")) ? testconfig.goConfig['testTags'] : testconfig.goConfig['buildTags'];
```

Comment:
We need to use single quotes, else the linter breaks the build.

Reply:
oops!